eternalcore-core/src/main/java/com/eternalcode/core/feature/home/command/HomeCommand.java

```diff
@@ -51,6 +53,7 @@ void execute(Player player) {
     }
 
     @Execute
+    @Required(1)
```

Comment:
`@ArgHome Home` powinien działać tak, że bierze home gracza jeśli go nie wpisał (czyli od min 0 do 1 max). Przydało by się usunąć już `@ArgHome Home` i zastąpić to normalnym argumentem `@Arg Home`.